resources/execute_event_handler.rb

```diff
@@ -0,0 +1,182 @@
+# frozen_string_literal: true
+
+resource_name :execute_event_handler
+provides :execute_event_handler
+
+# Resource to execute BYOS event handler
+
+property :event_name, String, name_property: true
+property :event_command, String, required: false
+
+default_action :run
+
+action :run do
+  if new_resource.event_command.nil?
+    Chef::Log.info("No command defined for Event #{new_resource.event_name}, noop")
+    return
+  end
+
+  event_log = '/var/log/parallelcluster/byos-plugin.log'
+  event_cwd = '/home/byos'
+  event_user = 'byos'
```

Comment:
can be done in a separate PR but we might want to set these 3 as shared vars in attributes?

Reply:
done in a separate commit